src/Impute.jl

```diff
@@ -324,32 +324,4 @@ julia> Impute.srs(df; rng=MersenneTwister(1234), context=Context(; limit=1.0))
 ```
 """ srs
 
-"""
-    svd!(data::AbstractMatrix; limit=1.0)
-
-Utility method for `impute!(data, :svd; limit=limit)`
-"""
-svd!(data::AbstractMatrix; limit=1.0) = impute!(data, :svd; limit=limit)
-
-"""
-    svd(data::AbstractMatrix; limit=1.0)
-
-Utility method for `impute(data, :svd; limit=limit)`
-"""
-svd(data::AbstractMatrix; limit=1.0) = impute(data, :svd; limit=limit)
-
-"""
-    knn!(data::AbstractMatrix; limit=1.0)
-
-Utility method for `impute!(data, :knn; limit=limit)`
-"""
-knn!(data::AbstractMatrix; limit=1.0) = impute!(data, :knn; limit=limit)
-
-"""
-    knn(data::AbstractMatrix; limit=1.0)
-
-Utility method for `impute(data, :knn; limit=limit)`
-"""
-knn(data::AbstractMatrix; limit=1.0) = impute(data, :knn; limit=limit)
-
```

Comment:
Would have been nice if these were first deprecated but a breaking release is going to be breaking.

Reply:
Yeah, I don't remember why they were separate. I think the svd PR was opened before the deprecation and I forgot to update that. I think the knn PR just copied what the svd PR did. Regardless, these methods would have still thrown a deprecation warning because they're still calling the symbol method with a limit keyword.